addons/io_scene_gltf2/blender/imp/gltf2_blender_scene.py

```diff
@@ -43,6 +43,7 @@ def create(gltf):
 
         if bpy.context.mode != 'OBJECT':
             bpy.ops.object.mode_set(mode='OBJECT')
+        BlenderScene.select_imported_objects(gltf)
         BlenderScene.set_active_object(gltf)
```

Comment:
Might be worth including a comment here, indicating that we're doing this for consistency with addons X, Y, and Z.

Reply:
It seems natural enough to me.

Reply:
If we got a bug report, "Importing glTF model loses my previous selection, please don't", I guess I'd want to remember why we do this — _Not_ selecting the imported objects also seemed natural enough to me, until learning it was a convention already. ;)

Reply:
Do you want me to add a comment? You could already have git blamed it, I put the reason in the commit message.

The reason its natural is not just the other importers, even just adding a Cube has this behavior. Everything else is deselected and only the Cube is selected.

Reply:
Git blame disappears when code moves around — I tend to prefer having real comments on anything that is likely to be unintuitive to someone who didn't write it. But if you feel this is intuitive enough, we can skip the comment here.